components/prize_pool/commons/prize_pool.lua

```diff
@@ -600,12 +600,58 @@ function PrizePool:_storeLpdb()
 		lpdbEntry.players = mw.ext.LiquipediaDB.lpdb_create_json(lpdbEntry.players or {})
 		lpdbEntry.extradata = mw.ext.LiquipediaDB.lpdb_create_json(lpdbEntry.extradata or {})
 		local lowerCaseParticipant = mw.ustring.lower(lpdbEntry.participant)
-		mw.ext.LiquipediaDB.lpdb_placement('ranking_' .. prizePoolIndex .. '_' .. lowerCaseParticipant, lpdbEntry)
+
+		if self.options.storeLpdb then
+			mw.ext.LiquipediaDB.lpdb_placement('ranking_' .. prizePoolIndex .. '_' .. lowerCaseParticipant, lpdbEntry)
+		end
+
+		if self.options.storeSmw then
+			self:_storeSmw(lpdbEntry)
+		end
 	end
 
 	return self
 end
 
+function PrizePool:_storeSmw(lpdbEntry)
+	-- TODO: Once Opponents has been added (instead of current Partipants), this can be cleaned up nicely
+	local isParty = type(lpdbEntry.players) == 'table' and Table.isNotEmpty(lpdbEntry.players)
+	local isTeam = String.isNotEmpty(lpdbEntry.participanttemplate)
+
+	local smwData = {
+		['has tournament page'] = lpdbEntry.parent,
+		['has tournament name'] = lpdbEntry.tournament,
+		['has tournament type'] = lpdbEntry.type,
+		['has tournament series'] = lpdbEntry.series,
+		['is result type'] = lpdbEntry.mode,
+		['has game'] = lpdbEntry.game,
+		['has date'] = lpdbEntry.date,
+		['is tier'] = lpdbEntry.liquipediatier,
+		['has placement'] = lpdbEntry.placement,
+		['has prizemoney'] = lpdbEntry.prizemoney,
+		['has last opponent'] = lpdbEntry.lastvs,
+		['has last score'] = lpdbEntry.lastscore,
+		['has last opponent score'] = lpdbEntry.lastvsscore,
+		['has last wdl'] = lpdbEntry.groupscore,
+		['has weight'] = lpdbEntry.weight,
+	}
+
+	if isParty then
+		smwData['has team page'] = lpdbEntry.players.p1team
+		smwData['has team'] = lpdbEntry.players.p1team
+		smwData['has player id'] = lpdbEntry.participant
+		smwData['has player page'] = lpdbEntry.participantlink
+		smwData['has flag'] = lpdbEntry.participantflag
+
+	elseif isTeam then
+		smwData['has team page'] = lpdbEntry.participant
+	else
+		smwData['has literal team'] = lpdbEntry.participant
+	end
+
```

Comment:
some option to customize would be good imo
there are wikis that have several additional properties they would want to set